examples/DynamicCantileverCase/dynamic_cantilever_helper.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
+
+
+class DynamicCantileverVibration:
+    def __init__(
+        self,
+        base_length,
+        base_area,
+        moment_of_inertia,
+        youngs_modulus,
+        density,
+        mode,
+        end_velocity=0.0,
+    ):
+        self.base_length = base_length
+        self.end_velocity = end_velocity
+
+        assert isinstance(mode, int)
+        assert mode >= 0
+        assert mode < 5
+        self.mode = mode
+
+        # First four roots to
+        # cosh(beta * base_length) * cos(beta * base_length) + 1 = 0
+        #
```

Comment:
Underneath the equation you can post the solution for beta, so it is more clear where are they from.

Reply:
[144e2af](https://github.com/GazzolaLab/PyElastica/pull/173/commits/144e2afdb705bd31365ba618f447211d23619aed)